### PR TITLE
speech:add ga samples

### PR DESCRIPTION
**Pull request by munkhuushmgl**

### CODE REVIEW

**Comment by munkhuushmgl** [comment]
Looks like dialogflow tests are failing.

**Comment by amanda-tarafa** [comment]
Rerunning kokoro. I had to disable Dialogflow, see #947

**Comment by amanda-tarafa** [comment]
And it's green.

File: speech/api/TranscribeContext/Program.cs [hunk 1]
```diff
@@ -0,0 +1,41 @@
+﻿// Copyright 2018 Google Inc.
```
Comment (amanda-tarafa) [inline speech/api/TranscribeContext/Program.cs]: nit: 2020 :)

File: speech/api/TranscribeContext/TranscribeContextClass.cs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// [START speech_adaptation]
+
+using System;
+using Google.Cloud.Speech.V1;
```
Comment (amanda-tarafa) [inline speech/api/TranscribeContext/TranscribeContextClass.cs]: nit: sort alphabetically

File: speech/api/TranscribeContext/TranscribeContextClass.cs [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+namespace GoogleCloudSamples
+{
+    public class TranscribeContextClass
```
Comment (amanda-tarafa) [inline speech/api/TranscribeContext/TranscribeContextClass.cs]: nit: Consider renaming to TranscribeContextClasses (the file as well)

Comment (munkhuushmgl) [reply speech/api/TranscribeContext/TranscribeContextClass.cs]: due to overlapping method name, cannot assign that name. I will come up with differnet class name

Comment (munkhuushmgl) [reply speech/api/TranscribeContext/TranscribeContextClass.cs]: How about speechContextClasses

**Pull request merged**